### PR TITLE
Dovzhyk volodymyr hw 4

### PR DESCRIPTION
**Pull request by ReSpekTeR**

### CODE REVIEW

File: HW4/hw4_task_1.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+x=int(input("""Рік є високосним, якщо він кратний 4, але не кратний 100, або ж кратний 400
+Немаєте бажання рахувати? Я порахую за вас. 
+Введіть рік: """))
+
+intercalaryty='високосний' if x%4==0 and x%100!=0 or x%400==0 else 'звичайний'
+century=x//100 if x%100==0 else x//100+1
+print('Рік',x,'є',intercalaryty,'і належить',century,'століттю',)
+
```
Comment (katemalash) [inline HW4/hw4_task_1.py:8]: Nice Decision, but please remove these empty lines.

File: HW4/hw4_task_2.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import math
+forma = input('Ведіть назву фігури (трикутник, прямоукутник чи коло) ')
+if forma=='трикутник'or forma=='прямоукутник' or forma=='коло':
+    if forma == 'трикутник':
+        a=int(input('Ввeдіть довжину сторони a: '))
+        b=int(input('Ввeдіть довжину сторони b: '))
+        c=int(input('Ввeдіть довжину сторони c: '))
+        p = (a + b + c) / 2
+        s = math.sqrt((p * (p - a) * (p - b) * (p - c)))
```
Comment (katemalash) [inline HW4/hw4_task_2.py:9]: You know this formula! Yeah!

File: HW4/hw4_task_2.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        a = int(input('Ввeдіть довжину сторони a: '))
+        b = int(input('Ввeдіть довжину сторони b: '))
+        s = a*b
+    elif forma == 'коло':
+        r = int(input('Ввeдіть радіус: '))
+        s = math.pi*r**2
+    print("Площа для фігури",forma,'=',s)
+else:
+    print("Error: Не вмію рахувати для цієї фігури")
+
```
Comment (katemalash) [inline HW4/hw4_task_2.py:20]: Good Decision, but empty line.

File: HW4/hw4_task_3.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        digits='three'
+    elif digits == 4:
+        digits = 'four'
+    elif digits == 5:
+        digits = 'five'
+    elif digits == 6:
+        digits = 'six'
+    print('Number',num,'is',sign,'and consists of',digits,'digit(s)')
+else:
+    print('Oh, too many letters')
```
Comment (katemalash) [inline HW4/hw4_task_3.py:19]: This decision works but it would be good to improve it without such amount of "if"s

File: HW4/hw4_task_4.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+from math import sqrt
+r = float(input('Ведіть радіус кола :'))
+s = float(input('Ведіть площу залу :'))
+a = sqrt(s)  # визначаєм довжину сторони квадрата
+if r<a:
+    k = float(input('Введіть мінімальну відстань від стіни до сцени:'))
+    if a-r*2>k:
```
Comment (katemalash) [inline HW4/hw4_task_4.py:7]: Are you sure this is correct formula?

File: HW4/hw4_task_5.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+money = int(input('Ввeдіть суму, грн: '))
+k_200 = money//200
+k_100 = (money-k_200*200)//100
+k_10 = (money-k_200*200-k_100*100)//10
+k_1 = (money-k_200*200-k_100*100-k_10*10)
+
+print("200:",k_200,"\n100:",k_100,"\n10:",k_10,"\n1:",k_1)
```
Comment (katemalash) [inline HW4/hw4_task_5.py:7]: Good Decision.

File: HW4/hw4_task_6.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    else:
+        where='side'
+
+    if seat%2==0:
+         where+=' at the top'
+    else:
+         where+=' at the bottom'
+    print('Your place number №',seat,where)
+else:
+    print('Don\'t cheat')
```
Comment (katemalash) [inline HW4/hw4_task_6.py:17]: Good Decision.

**Review by katemalash — APPROVED** [review]
In general, home work is done good. You can merge pull-request after fixing some issues.

File: HW4/hw4_task_1.py (hunk 1 above)
Comment (ReSpekTeR) [reply HW4/hw4_task_1.py]: Done

File: HW4/hw4_task_2.py (hunk 3 above)
Comment (ReSpekTeR) [reply HW4/hw4_task_2.py]: Done

File: HW4/hw4_task_3.py (hunk 4 above)
Comment (ReSpekTeR) [reply HW4/hw4_task_3.py]: Done. Code was refactored

File: HW4/hw4_task_4.py (hunk 5 above)
Comment (ReSpekTeR) [reply HW4/hw4_task_4.py:7]: Yes. I'm sure it is.

Comment (ReSpekTeR) [reply HW4/hw4_task_4.py:7]: can replace this " if r<a: " to "if 2 * r<a: "

**Pull request merged**